content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -261,12 +261,17 @@ serialized to JSON as the body.
 Webhooks can specify what versions of `AdmissionReview` objects they accept
 with the `admissionReviewVersions` field in their configuration:
 
-```yaml
-apiVersion: admissionregistration.k8s.io/v1
-kind: ValidatingWebhookConfiguration
-webhooks:
-- name: my-webhook.example.com
-  admissionReviewVersions: ["v1", "v1beta1"]
+```json
+{
+  "apiVersion": "admissionregistration.k8s.io/v1",
+  "kind": "ValidatingWebhookConfiguration",
+  "webhooks": [
+    {
+      "name": "my-webhook.example.com",
+      "admissionReviewVersions": ["v1", "v1beta1"]
+    }
+  ]
+}
 ```
```

Comment:
I don't think we need this change. The other change, we might want.

Reply:
I thought the example in the **`Request`** needs to be changed as mentioned in this issue.

Reply:
Do I need to change this back to YAML?

Reply:
yes please

Reply:
Done the required changes PTAL!